usr/src/uts/common/fs/zfs/sys/zap.h

```diff
@@ -88,22 +89,12 @@ extern "C" {
 
 /*
  * Specifies matching criteria for ZAP lookups.
+ * MT_IGNORE_FORM - normalize the lookup according to FS settings.
```

Comment:
I don't think the ZAP is examining the FS settings, it's looking at the ZAP settings, specifically zap_normflags, which can be `U8_TEXTPREP_NF* | U8_TEXTPREP_TOUPPER`.  Could you expand the comment to include that?  In particular, that this can include both unicode normalization and case-insensitivity.  I wonder if this would be better named `MT_NORMALIZE`.